.changeset/thirty-owls-lay.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/core': patch
+'storybook': patch
```

Comment:
No need to include storybook in the changeset

Reply:
done